### PR TITLE
Add tests for `screenLeft` and `screenTop`

### PR DESCRIPTION
**Pull request by foolip**
Matches https://github.com/w3c/csswg-drafts/pull/2669.

Note that `screenX` and `screenY` themselves aren't tested at all:
https://github.com/web-platform-tests/wpt/issues/5471

### CODE REVIEW

**Review by emilio — APPROVED** [review]
The lack of testing for cssom-view is indeed concerning.

File: css/cssom-view/screenLeftTop.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+<script src="/resources/testharnessreport.js"></script>
+<script>
+test(() => {
+  assert_equals(typeof window.screenLeft, "number", "screenLeft type");
+  assert_equals(window.screenLeft, window.screenX, "alias of screenX");
+}, "screenLeft");
+
+test(() => {
+  assert_equals(typeof window.screenLeft, "number", "screenTop type");
+  assert_equals(window.screenLeft, window.screenX, "alias of screenY");
```
Comment (emilio) [inline css/cssom-view/screenLeftTop.html]: screenTop, screenY

**Review by emilio — APPROVED** [review]
Needs those changes though :)

File: css/cssom-view/screenLeftTop.html (hunk 1 above)
Comment (foolip) [reply css/cssom-view/screenLeftTop.html]: Haha, I fail at search-replace :)

**Pull request merged**